doc-content/enterprise-only/fuse/ba-karaf-xml-uninstall-proc.adoc

```diff
@@ -1,17 +1,26 @@
-[id='features-xml-removing']
-= Removing features XML files
-If your installation contains an older version of the {PRODUCT} features XML files (for example, `kie-karaf-features-<VERSION>-features.xml`), you must remove these files and all associated files before installing the most recent features XML files.
+[id='ba-karaf-xml-uninstall-proc']
+= Uninstalling obsolete {PRODUCT} features XML files on Karaf
+
+If your installation contains older versions of the {PRODUCT} features XML files (for example, `kie-karaf-features-<VERSION>-features.xml`), you must remove these files and all associated files before installing the most recent features XML files.
 
 .Prerequisites
-* Obsolete features XML files exist in your installation.
-COMMENT: How do I know if I have obsolete files? How do I know what features use the obsolete files?
+* Obsolete features XML files exist in your installation Apache Karaf installation.
+
 
 .Procedure
-. Enter the following command to start the {FUSE_LONG} console where COMMENT: What is INSTALLATION_DIRECTORY? The Fuse home? The Karaf home?
+. Enter the following commands to determine whether your installation contains obsolete {PRODUCT} features XML files:
++
+[source]
+----
+$ JBossFuse:karaf@root> feature:repo-list
+$ JBossFuse:karaf@root> feature:list
+----
+
+. Enter the following command to start the {FUSE_LONG} console where `FUSE_HOME` is the Fuse installation directory:
 +
 [source]
 ----
-$ ./INSTALLATION_DIRECTORY/bin/fuse
+$ ./FUSE_HOME/bin/fuse
 ----
 . Enter the following command to uninstall features or applications that use obsolete features XML files, where `<FEATURE_NAME>` is the name of the feature that you want to uninstall.
 +
```

Comment:
You use the variable format "`<FEATURE_NAME>`" here but just "`FUSE _HOME`" above and then "`BUNDLE_ID`" later. However you do it most often in your docs, do that I guess. We're all pretty inconsistent on this anyway, and it's on the list in that CCS-wide style guide thing, but meanwhile, I think consistency throughout the doc at least. For me, I think I almost always use < >. I see that whoever last updated the clustering doc (Clifton I think) does the same, even with `<EAP_HOME>/bin/...`. 

Below, you say "the following examples demonstrate" and then the next step "show". Maybe use the same time.

Also, typo in "The following examples show" (make plural, or change to "shows"), and end in colon "components:"

IMO, I'd make them singular anyway, "following example shows...". Clearer and more consistent because same for each example as well.